src/client/styles/components/_summary.scss

```diff
@@ -38,3 +51,16 @@
     @include core-font(16);
   }
 }
+
+.summary caption{
```

Comment:
Is this element not the same as `.summary__title`? These declarations should be applied there if there are at all necessary.